boards/mcb2388/include/periph_conf.h

```diff
@@ -83,9 +83,18 @@ static const spi_conf_t spi_config[] = {
         .pinsel_msk_miso = (BIT14 | BIT15), /* P1.23 */
         .pinsel_msk_clk  = (BIT8  | BIT9),  /* P1.20 */
     },
+    {
+        .dev = SPI1,
+        .pinsel_mosi = 0,
+        .pinsel_miso = 0,
+        .pinsel_clk  = 0,
+        .pinsel_msk_mosi = (BIT19), /* P0.9 */
+        .pinsel_msk_miso = (BIT17), /* P0.8 */
+        .pinsel_msk_clk  = (BIT15), /* P0.7 */
+    },
 };
 
-#define SPI_NUMOF           (1)
+#define SPI_NUMOF           (2)
```

Comment:
```suggestion
#define SPI_NUMOF           ARRAY_SIZE(spi_config)
```
?

Reply:
Yea here it would actually work.
But the drivers that use interrupts all have constructs [like](https://github.com/RIOT-OS/RIOT/blob/master/cpu/lpc2387/periph/uart.c#L38) [this](https://github.com/RIOT-OS/RIOT/blob/master/cpu/lpc2387/periph/i2c.c#L39) where `ARRAY_SIZE` will fail.

So I wanted this to stay consistent with the rest of the file.

Reply:
Using `ARRAY_SIZE()` whenever possible is also consistent; at least from the meta perspective.

Reply:
Meh, now all the tings that expect `SPI_NUMOF` to be an `int` do fail

Reply:
No, this is the issue:

> `implicit declaration of function 'ARRAY_SIZE' [-Werror=implicit-function-declaration]`

It is in `kernel_defines.h`, if I recall correctly.

Reply:
Oops, you are right.